lib/providers/openstreetmap.js

```diff
@@ -175,4 +176,96 @@ var OpenStreetMap = Provider.extend(function (self, opts) {
     };
 });
 
+OpenStreetMap.fixture = function(opts) {
+    /**function:OpenStreetMap.fixture(opts)
+
+    Returns an HTTP resource fixture for an OpenStreetMap location query.
+
+    :param string opts.query:
+        The address that is to be queried. Required.
+
+    :param array_of_strings opts.address_list:
+         An array of the list of `formatted_address`'s that should be sent in
+         the response. If response_data is included, this will be ignored.
+         Defaults to ``[]``.
+
+    :param boolean hard_boundary:
+        Whether the request should limit results to a fixed bounding box.
+        Defaults to `true`.
+
+    :param integer address_limit:
+        The maximum number of search results that should be requested. Defaults
+        to `30`.
```

Comment:
Does Sphinx allows single backticks for some values? Just want to make sure it doesn't before fixing those.